src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -52,89 +56,94 @@ export const SchemaRow: React.FunctionComponent<SchemaRowProps> = ({ schemaNode,
 
   const isBrokenRef = typeof refNode?.error === 'string';
 
+  const rootLevel = renderRootTreeLines ? 1 : 2;
   const childNodes = React.useMemo(() => calculateChildrenToShow(typeToShow), [typeToShow]);
   const combiner = isRegularNode(schemaNode) && schemaNode.combiners?.length ? schemaNode.combiners[0] : null;
```

Comment:
this is wrong, but we can fix it some other day.
There might be multiple combiners :grimacing:, but I'm not sure how we'd like to render that.

Reply:
Yesss I did notice a number of other weird things as I was going through this - but I already got pulled wayyy too far into the rabbit hole with the changes here already so 🤷‍♂️.